sampler.js

```diff
@@ -1,4 +1,35 @@
 var fs   = require('fs');
+// { setup section
+var OUTPUT = './output';
+// create the _output_ folder if it does not exist
+try {fs.accessSync(OUTPUT)} catch(e) {
```

Comment:
I'm thinking if it weren't better to either:
* add a `.gitkeep` to the directory
* let sampler output to the console